addon/components/dashboard/calendar.hbs

```diff
@@ -87,6 +87,7 @@
                           @checked={{includes type.id @selectedSessionTypeIds}}
                           @add={{fn @addSessionTypeId type.id}}
                           @remove={{fn @removeSessionTypeId type.id}}
+                          @id={{type.id}}
```

Comment:
I'm a little worried that this being called `id` is going to make a future problem where I decide every checkbox needs an ID and use this passed property for it, however it's not unique. Maybe do `@id="seession-type-{{type.id}}"` (and the other places this gets set) or you could set `data-test-filter-checkbox-id={{type.id}}` directly here and ensure `...attributes` gets called on the `<Dashboard::FilterCheckbox` component.

Reply:
i renamed the argument to `@targetId` and the `data-test-` attribute to `data-test-filter-checkbox-target-id`. i hope this passes muster now.